Exercises/5-range-odd.js

```diff
@@ -4,7 +4,7 @@
 const rangeOdd = (begin, stop) => {
   const array = [];
   for (let i = begin; i <= stop; i++) {
-    (i % 2) ? array.push(i) : null;
+    if (i % 2 !== 0) ? array.push(i);
```

Comment:
It seems to me:
if (i % 2 !== 0) array.push(i);
and 
if (i % 2) array.push(i);
are just the same, no? Why the first one is better?